environments/prod/prod.tfvars

```diff
@@ -3118,6 +3118,15 @@ frontends = [
         match_variable = "RequestCookieNames"
         operator       = "Equals"
         selector       = "sptribs-cookie-preferences"
+        }, {
```

Comment:
looks off

Reply:
repushed, don't know what it did there...